modules/oe_whitelabel_search/tests/src/Kernel/SearchBlockTest.php

```diff
@@ -94,27 +94,51 @@ public function testNavigationRightSearchBlockRendering(): void {
     $render = $this->container->get('renderer')->renderRoot($build);
     $crawler = new Crawler($render->__toString());
 
-    // Assert the form rendering.
-    $form = $crawler->filter('form');
+    // Search wrapper div.
+    $wrapper = $crawler->filter('div.search-dropdown.dropdown');
+    $this->assertCount(1, $wrapper);
+
+    // Toggle button.
+    $toggle = $wrapper->filter('button.oe-whitelabel-search-form');
+    $this->assertCount(1, $toggle);
+    $this->assertSame('block-oe-whitelabel-search-form', $toggle->attr('id'));
+    $this->assertStringContainsString('oe-whitelabel-search-form dropdown-toggle d-xl-none px-2 btn btn-ghost btn-md', $toggle->attr('class'));
+
+    // Icon inside toggle.
+    $icon = $toggle->filter('svg.bi.icon--fluid');
+    $this->assertCount(1, $icon);
+
+    // Dropdown menu container.
+    $dropdown = $wrapper->filter('div.dropdown-menu');
+    $this->assertCount(1, $dropdown);
+    $this->assertStringContainsString('dropdown-menu px-3 px-xl-0 d-xl-block ms-xl-4', $dropdown->attr('class'));
+
+    // The form.
+    $form = $dropdown->filter('form');
     $this->assertCount(1, $form);
-    $this->assertSame('oe-whitelabel-search-form', $form->attr('id'));
```

Comment:
We used to assert the id, then you changed that to filter by id, now we don't assert or filter by id anymore.